state/VehiclesStoreProvider/VehiclesStoreProvider.tsx

```diff
@@ -74,6 +77,20 @@ const VehiclesStoreProvider = ({ children }: PropsWithChildren) => {
     [insertMutation, vehicles],
   )
 
+  const editVehicle = useCallback(
+    async (vehicle: EditVehicle) => {
+      await updateMutation.mutateAsync({
+        id: vehicle.id,
+        updateVehicleDto: {
+          vehiclePlateNumber: vehicle.licencePlate,
+          name: vehicle.vehicleName,
+          isDefault: !!vehicle.isDefault,
```

Comment:
It will probably be changed according to PATCH request, but in general, we want to avoid unnecessary type "changes" as this. There is no need to use `!!` (cast it to boolean) (`vehicle.isDefault` can be `true`, `false` or `undefined` an it is completely okay because the same type is accepted by the endpoint).